developer/python-django-devtools-reference.rst

```diff
@@ -0,0 +1,229 @@
+Python/django dev tools reference
+=================================
+
+We aim at helping technical users and contributors to work on OpenWISP
+and we introduce them to some python tools they can find very useful
+during development and debugging.
+
+Python and its advantages
+-------------------------
+
+As you may already know, python is a programming language, more
+specifically an interpreted high-level programming language for
+general-purpose programming. It is also widely used these days!
+Big companies like Google, Disney and Dropbox make excessive use
+of this programming language! You'll know why in an instant.
+
+- To begin with, the syntax in Python helps programmers to do
+  coding in fewer steps as compared to other languages such as
+  Java or C++ because it uses relatively less lines and words
+  while still performing the same operations.
+- Additionally, python offers excellent readability and uncluttered
+  simple-to-learn syntax which helps beginners to utilize this
+  programming language.
+- Also, python has a large standard library full of pre-defined
+  functions which can help fulfil tasks related to string operations,
+  Internet, web service tools, operating system interfaces and protocols
+  a lot more quickly and easily.
+- Believe it or not, python language is developed under an OSI-approved
+  open source license, which makes it free to use and distribute,
+  including for commercial purposes!
+- Python has built-in list and dictionary data structures which can be
+  used to construct fast runtime data structures too!
+- Python has clean object-oriented design, provides enhanced
+  process control capabilities, and possesses strong integration and
+  text processing capabilities and its own unit testing framework,
+  all of which contribute to the increase in its speed and productivity.
+- Finally, finding developers who know python is very easy and allows for
```

Comment:
these 2 points are important, they deserve one bullet each (finding developers and networking)